app/assets/javascripts/darkswarm/directives/country_selector.js.coffee

```diff
@@ -0,0 +1,11 @@
+Darkswarm.directive "countrySelector", () ->
```

Comment:
Ah, yes, it's currently not processed by Ruby. But you can add the `.erb` extension to do that. One example is `app/assets/javascripts/darkswarm/services/geo.js.erb.coffee`. That will be executed during compile time. So if you change the default country, you need to rebuild the assets.